db-checkbook/build_scripts/01_dataloading.py

```diff
@@ -1,16 +1,39 @@
-from dcpy.connectors.s3 import client
+#from dcpy.connectors.s3 import client
+import json
+import os
+from functools import wraps
+from pathlib import Path
+
+import pandas as pd
+import yaml
+import geopandas as gpd
+
+
+
+#def do_stuff() -> None:
+ #   print("doing stuff")
+  #  s3_client = client()
+   # available_buckets = [
+    #    bucket["Name"] for bucket in s3_client.list_buckets()["Buckets"]
+    #]
+   # print(f"This S3 client is a {type(s3_client)}")
+   # print(f"This S3 client has access to buckets: {available_buckets}")
+
+
+# reading in the checkboob daa
+def read_from_csv(name: str) -> pd.DataFrame:
```

Comment:
maybe we should specify the url on digital ocean as a param as well so this function can be generalized to different csvs (noting that the url is currently a placeholder)

Reply:
A couple notes on aligning this with our bash functionality in terms of how its used
- store the "base" url as a literal (i.e. at top of file for now, have `recipes_url="https://...` or something like that
- use name to format string in read_csv
- have `version` as an optional arg with default value `latest`. In this case at the moment, the version is `20210122`